src/test/java/com/flightstats/hub/webhook/WebhookStateReaperTest.java

```diff
@@ -30,6 +32,7 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+@Execution(ExecutionMode.SAME_THREAD)
```

Comment:
A few of our tests run over each other's state if run in parallel, i.e. trying to create the same node in zookeeper at the same time as another. This execution mode overrides the default parallel run setting.

Reply:
Cool.  I've definitely run across some classes that can't be internally parallelized.  Nice feature.